### ISSUE TITLE
Can't pick up pals in base after character transfer

### ISSUE THREAD

**Issue by insanebugbear** [jmkl009/PalworldCharacterTransfer#16]
After doing a character transfer, the guilds seem bugged. I can't pick up a pal or get the menu to show up to feed them. I think the character's unnamed guild isn't deleted causing some kind of strange problem.

**Comment by jmkl009** [comment]
If you are also doing a world/base transfer,  did  you check the "keep old guild ID after transfer" box at the character transfer stage?

**Comment by insanebugbear** [comment]
Yes, I had that toggled, I was transferring a character from one ID to another onto a copy of the same save. I've tried leaving and rejoining the guild but that seems to only be a temporary fix, after the server resets the bug returns.

**Comment by insanebugbear** [comment]
hmm okay I think I've fixed it for now, had to leave guild, rejoin and then drop a palbox and this seems to have fixed the prob, will update if it resets again

**Comment by insanebugbear** [comment]
okay never mind, every time the server resets the bug comes back, very unfortunate
